src/configs/index.js

```diff
@@ -58,5 +58,10 @@ export const config = {
     email:
       '^([\\w-]+(?:\\.[\\w-]+)*)@((?:[\\w-]+\\.)*\\w[\\w-]{0,66})\\.([a-z]{2,6}(?:\\.[a-z]{2})?)$',
     password: '^(?!.* )(?=.*[0-9])(?=.*[A-Z]).{8,30}$'
+  },
+  paginationPayload: {
+    skip: 0,
+    limit: 5,
+    productsPerPage: 5
```

Comment:
We have 3 news per row. It will be nicer if it will be 3/6/9. I think 6 will be alright

Reply:
Changed!